### PR TITLE
changes to FAQ + Community pages: remove Made With A-Frame Tumblr, link to A-Frame Blog, grammar changes

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: src/community/index.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ layout: page
...
+* [A-Frame Blog](https://aframe.io/blog/) - Stay up to date with the latest A-Frame changes and projects.
+* [**awesome-aframe** Repo](https://github.com/aframevr/awesome-aframe) - Peruse a list of awesome things created for or using A-Frame.
+* [Stack Overflow](https://stackoverflow.com/questions/tagged/aframe) - Ask questions and learn on Stack Overflow.
 
 ## WebVR
 
 WebVR is not just A-Frame. See what is happening in the greater community.
 
-* [WebVR Community Slack Channel](http://webvr.slack.com/) - [Sign up](https://webvr-slack.herokuapp.com/).
+* WebVR Community Slack Channel - [Sign up](https://webvr-slack.herokuapp.com/) to join the WebVR community on Slack.
```
Comment (ngokevin) [inline src/community/index.md]: The last part is a bit redundant

WebVR Community Slack Channel ... WebVR community on Slack

File: src/community/index.md [hunk 2]
```diff
@@ -8,17 +8,17 @@ layout: page
...
 
-* [A-Frame GitHub Repo](https://github.com/aframevr/aframe/) - Grab the code, file issues, and contribute.
+* [A-Frame GitHub Repo](https://github.com/aframevr/aframe) - Grab the code, file issues, and contribute.
 * A-Frame Community Slack Channel - [Sign up](https://aframevr-slack.herokuapp.com/) to share your work and join the discussion.
 * [A-Frame Twitter](https://twitter.com/aframevr) - Tweet at us.
-* [**Made With A-Frame** Tumblr](http://aframevr.tumblr.com/) - Check out a showcase of recent work done in A-Frame.
-* [**awesome-aframe** Repo](https://github.com/aframevr/awesome-aframe) - Go over a list of awesome things created regarding A-Frame.
-* [Stack Overflow](http://stackoverflow.com/questions/tagged/aframe) - Ask questions and learn on Stack Overflow.
+* [A-Frame Blog](https://aframe.io/blog/) - Stay up to date with the latest A-Frame changes and projects.
+* [**awesome-aframe** Repo](https://github.com/aframevr/awesome-aframe) - Peruse a list of awesome things created for or using A-Frame.
```
Comment (ngokevin) [inline src/community/index.md]: Maybe just ```with A-Frame```

File: src/community/index.md [hunk 3]
```diff
@@ -8,17 +8,17 @@ layout: page
...
 A-Frame grown into a large active community by putting VR content creation into everyone's hands.
 
-* [A-Frame GitHub Repo](https://github.com/aframevr/aframe/) - Grab the code, file issues, and contribute.
+* [A-Frame GitHub Repo](https://github.com/aframevr/aframe) - Grab the code, file issues, and contribute.
 * A-Frame Community Slack Channel - [Sign up](https://aframevr-slack.herokuapp.com/) to share your work and join the discussion.
 * [A-Frame Twitter](https://twitter.com/aframevr) - Tweet at us.
-* [**Made With A-Frame** Tumblr](http://aframevr.tumblr.com/) - Check out a showcase of recent work done in A-Frame.
-* [**awesome-aframe** Repo](https://github.com/aframevr/awesome-aframe) - Go over a list of awesome things created regarding A-Frame.
-* [Stack Overflow](http://stackoverflow.com/questions/tagged/aframe) - Ask questions and learn on Stack Overflow.
+* [A-Frame Blog](https://aframe.io/blog/) - Stay up to date with the latest A-Frame changes and projects.
```
Comment (ngokevin) [inline src/community/index.md]: Possibly more general ```Stay up to date with what's happening with A-Frame```

**Comment by ngokevin** [comment]
r+wc

File: src/community/index.md (hunk 1 above)
Comment (cvan) [inline src/community/index.md]: will do

**Pull request merged**